backend/services/implementations/__tests__/testService.test.ts

```diff
@@ -0,0 +1,54 @@
+import TestService from "../testService";
+
+import db from "../../../testUtils/testDb";
+import { QuestionType } from "../../../models/test.model";
+
+const questions = [
+  {
+    questionType: QuestionType.NUMERIC_ANSWER,
+    questionPrompt: "Question",
+    questionMetadata: {
+      answer: 3,
+    },
+  },
+];
```

Comment:
could we move these `testUtils` - seems like we're doing that for some of our other tests